syslog_rfc5424_formatter.py

```diff
@@ -36,24 +36,17 @@ class RFC5424Formatter(logging.Formatter, object):
     the format string that you pass in the constructor is only
     applied to the message body (and should typically just be %(message)).
 
-    The '- -' sections in the resulting message are the "msg ID" and
-    "Structured-Data" Elements, respectively
-
-    MSGID (Description from RFC5424):
-       The MSGID SHOULD identify the type of message.  For example, a
-   firewall might use the MSGID "TCPIN" for incoming TCP traffic and the
-   MSGID "TCPOUT" for outgoing TCP traffic.  Messages with the same
-   MSGID should reflect events of the same semantics.  The MSGID itself
-   is a string without further semantics.  It is intended for filtering
-   messages on a relay or collector.
-   The NILVALUE SHOULD be used when the syslog application does not, or
-   cannot, provide any value.
-
    Stuctured Data Example:
         [exampleSDID@32473 iut="3" eventSource="Application" eventID="1011"]
     '''
-    def __init__(self, *args, **kwargs):
+    def __init__(self, appname='python', procid='-', msgid='-', data='-', *args, **kwargs):
```

Comment:
there should be a fallback to make `name` continue being the logger name

perhaps if `appname` is None (and default it to None)?

Reply:
I can I just didn't know what name was being used as this field should identify APP-NAME which "root" wasn't really doing. To keep backwards compatibility I can do this.

Reply:
well, "root" is just if you log with the root logger.

The expected case is that you would name your logger something sensible. 😄

Reply:
OK I have removed appname as no reason to have 2 ways to set it.